dfffrontend/index.html

```diff
@@ -151,6 +159,20 @@ <h2 class="invoice-title">Upload kroz CSV</h2>
             </div>
         </div>
     </div>
+
+
+    <script>
```

Comment:
we want this code to be in the js files and not directly in html. You can place it in invoicemanualentry.js
On script load, you can read localstorage and set the input, and once the signatures have been succesfully submitted for checks you can store the api key in local storage because you know that it is a correct one at that point.

Reply:
Relocated the code to the right places.